m4_sessions/1711-inning/Group_2.md

```diff
@@ -1,9 +1,19 @@
 Group Two
 
 Session Title: Description: Why should students come to this session?
+Because Turing is designed around failure and how to learn from that failure. This class seeks to teach how to more effectively learn from that struggle.
 
 Session Outcomes: What will students leave with or gain from this session?
+How to see failure as a stepping stone to future successes.
```

Comment:
There are multiple past Gear Ups to take a look at: https://github.com/turingschool/gear-up/blob/master/m4_sessions/1708-inning/failing_gracefully.md

https://github.com/turingschool/gear-up/blob/master/struggle.markdown

Take a look at both of these to gather even more ideas about your outline.